cluster/src/main/java/com/hartwig/pipeline/CommandLineOptions.java

```diff
@@ -1,17 +1,12 @@
 package com.hartwig.pipeline;
 
-import java.util.Optional;
-
-import org.apache.commons.cli.CommandLine;
-import org.apache.commons.cli.DefaultParser;
-import org.apache.commons.cli.HelpFormatter;
-import org.apache.commons.cli.Option;
-import org.apache.commons.cli.Options;
-import org.apache.commons.cli.ParseException;
+import org.apache.commons.cli.*;
 import org.jetbrains.annotations.NotNull;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Optional;
+
 public class CommandLineOptions {
```

Comment:
I think there must be some unused methods and constants in this class now that you've removed MODE. I also thing you need to remove it from the options list.

Reply:
Bah. Good catch.